lib/src/native/realm_core.dart

```diff
@@ -1715,7 +1715,7 @@ extension on Pointer<Void> {
 }
 
 extension on Pointer<Utf8> {
-  String? toRealmDartString({bool treatEmptyAsNull = false, int? length, bool freeNativeMemory = false}) {
+  String? toRealmDartString({bool treatEmptyAsNull = false, int? length, bool freeRealmMemory = false}) {
```

Comment:
What is the reasoning behind this name change? I kind of preferred the old name.

Reply:
the reason is we are doing `realm_free` instead of other kind of  native memory free. So I renamed it to be more precise. Should I revert it?